bentoml/service.py

```diff
@@ -553,6 +574,21 @@ def artifacts(self):
     def env(self):
         return self._env
 
+    @property
+    def web_static_content(self):
+        return self._web_static_content
+
+    @property
+    def get_web_static_content_path(self):
+        if self._bento_service_bundle_path and self.name:
```

Comment:
This should be `if self.web_static_content and self._bento_service_bundle_path` to make BentoService that does not have `web_static_content` to show an index page properly right?

self.name should already be presented

Reply:
> This should be `if self.web_static_content and self._bento_service_bundle_path` to make BentoService that does not have `web_static_content` to show an index page properly right?

Yeh, sorry missed that in the final commit

> self.name should always be presented

I swear previously I was getting an error on `self.name` but now it's working😅
I think I may have used `_bento_service_name` while testing and cause that was giving error I replaced everything with `self.name` and then never checked it properly.